File: Python/electronic.py
# electronic.py
from item import Item

class Electronic(Item):
    def __init__(self, id, nama, harga, merk, stok, garansi):
        super().__init__(id, nama, harga)
        self.merk = merk
        self.stok = stok
        self.garansi = garansi

    def to_row(self):
        return [self.id, self.nama, str(self.harga), self.merk, str(self.stok), str(self.garansi)]

File: JAVA/Electronic.java
// Electronic.java
public class Electronic extends Item {
    protected String merk;
    protected int stok;
    protected int garansiBulan;

    public Electronic(String id, String nama, int harga, String merk, int stok, int garansiBulan) {
        super(id, nama, harga);
        this.merk = merk;
        this.stok = stok;
        this.garansiBulan = garansiBulan;
    }

    @Override
    public String[] toRowParts() {
        return new String[]{id, nama, String.valueOf(harga), merk, String.valueOf(stok), String.valueOf(garansiBulan)};
    }
}
